UPGRADE-14.0.md

```diff
@@ -1040,3 +1040,9 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
     -   after migration to Next Image component there were some places left to adjust proper image sizes
     -   Fix additionally downloaded image is loaded after product page render. `Lightgallery` is not cooperating right with `NextImage` component. It doesn't know about which image (`width`) was loaded from `srcset` so it takes src from image loaded with `NextImage` which is full size of 3840px and loads it immediately after page loads in order to having it downloaded after user opens an image modal. After the image modal opens `LightGallery` loads original image (which is without `width` query) anyway. So preloaded image is loaded for nothing. Proper fix is replacing `Lightgallery` with new component for Image Modal Gallery.
     -   Remove `image-rendering:-webkit-optimize-contrast` from images. This property is causing some of images to be rendered with sharp edges on places where it is not wanted. Browsers algorithms seem to handle this value correctly on its own without need to use specific rendering property.
+
+-   replace Lightgallery with custom ModalGallery component ([#2995](https://github.com/shopsys/shopsys/pull/2995))
+    -   new features:
+        -   custom useKeyPress hook, responsible for handling key press events
+        -   new library for handling swiping events (react-swipeable)
+        -   gallery is loaded dynamically, allowing us to lower the size of product page, user don't even need to open gallery so what to not load it dynamically now...?
```

Comment:
I am not sure if this thought is finished? There are 3 dots, and it seems like not completely done.

Reply:
Three dots where meant to be there just because of form of the sentence, but I removed unnecessary comment